feat: CSS Cafe Menu (#41797)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-083.md

---
id: 5f45b45d099f3e621fbbb256
title: Part 83
challengeType: 0
dashedName: part-83
---

# --description--

> You change properties of a link when the link is actually clicked by using a <dfn>pseudo-selector</dfn> that looks like `a:active { propertyName: propertyValue; }`.

Reviewer: ```suggestion
You change properties of a link when the link is actually being clicked by using a <dfn>pseudo-selector</dfn> that looks like `a:active { propertyName: propertyValue; }`.
```

not sure if this is grammatically more precise. it just made more sense to me.

> Change the color of the footer `Visit our website` link to be `white` when a clicks on it.

Reviewer 2: ```suggestion
Change the color of the footer `Visit our website` link to be `white` when clicked on.
```

# --hints--

> You should use the `a:active` pseudoselector.

Reviewer 2: ```suggestion
You should use the `a:active` pseudo-selector.
```
